### ISSUE TITLE
Editor: A better warning is needed when a multi-instance task has no instance queries specified

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#223]
```
WHAT STEPS WILL REPRODUCE THE PROBLEM?

Open specification "Editor2UserManualExample.yawl"

Enter marquee selection mode

Select "Do Private Study", delete it.

Add Multiple composite task to replace it.
(Create multiple composite task, drawing lines as before and Select task
decomposition Do_Private_Study)

Select Instance detail,
Set Minimum Instances to 2
Maximum instances to 10
Continuation Threshold to 5
Instance creation to Static

Try to save the changed specification.

The following dialog appears:
Export File Generation Error
File save process resulted in a 'null' specification.
File not created.


WHAT IS THE EXPECTED OUTPUT? WHAT DO YOU SEE INSTEAD?

I expected to be able to save the specification with the changes. 

WHAT VERSION OF THE PRODUCT ARE YOU USING? ON WHAT OPERATING SYSTEM?

Version run:
YAWL4Enterprise_2.0_beta_windows_installer.exe
Downloaded 2008-11-17 [Update 3 candidate]
YAWL Editor, version: Build Date: 2008.11.10 18:32:29
YAWL Control Centre, version: UNKNOWN
YAWL Engine, version: UNKNOWN

Windows Server 2003 Enterprise Edition 40GB - VMware Workstation

```

Original issue reported on code.google.com by `sigfr...@dsv.su.se` on 17 Nov 2008 at 1:01

Attachments:
* [Editor2UserManualExample.yawl](https://storage.googleapis.com/google-code-attachments/yawl/issue-223/comment-0/Editor2UserManualExample.yawl)

**Comment by yawlfoundation** [comment]
```
This will happen if no instance queries are defined for the multi-instance task 
(and
has always been the case). If a verification is run, the problem is reported - 
but
the message is a bit cryptic. I'll leave this issue up and address the error 
msg in
the next release (after update3). 
```

Original comment by `yawl.mic...@gmail.com` on 18 Nov 2008 at 1:44
* Changed title: **Editor: A better warning is needed when a multi-instance task has no instance queries specified**
* Added labels: **Component-Editor**

**Comment by yawlfoundation** [comment]
Original comment by `yawl.mic...@gmail.com` on 18 Nov 2008 at 1:45
* Changed state: **Accepted**

**Comment by yawlfoundation** [comment]
```
I can understand if this is an error and there is definitely a better error 
message
needed for it.

But I cannot understand why it should not be possible to save the file anyway. 
That
is, why not just display a better error message, add to it that the 
specification
cannot be used by the engine but give an option to save it anyway.

After all you may have to leave your not yet completed specification for some 
reason.
Should you not be able to then save the work you have done but have to discard 
the
work you have done?

Or perhaps, I just want to sketch something to show a consultant that I want to
complete this work because I myself know nothing about "instance queries" etc 
myself
and I am not interested in finding out about it because I am a manager that 
have no
time to read the manual and guess to the meaning of "instance queries etc. 
Shall I
then not be able to save my sketch?? 

It is like Word not allowing you to save a document because there are some
grammatical errors in it. 
```

Original comment by `sigfr...@dsv.su.se` on 18 Nov 2008 at 7:37

**Comment by yawlfoundation** [comment]
```
resolved in editor 3.0 (mi variable support completely reworked)
```

Original comment by `yawl.mic...@gmail.com` on 9 Dec 2013 at 2:35
* Changed state: **Fixed**
